action.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Is Website vulnerable'
+description: 'Scans a url for public javascript library vulnerabilities'
+inputs:
+  scan-url:
+    description: 'Website url to scan'
+    required: true
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

Comment:
@mattorb is it enough to specify `Dockerfile` like that for the github action to take it from the root directory of the project or does this need to be something like `../Dockerfile` ?

Reply:
Yes, it is enough.   Github looks for that Dockerfile in the root of the repo by default.

Reply:
@mattorb cool, thanks!